test/CMakeLists.txt

```diff
@@ -50,6 +50,8 @@ function(add_spvtools_unittest)
     target_include_directories(${target} PRIVATE
       ${SPIRV_HEADER_INCLUDE_DIR}
       ${spirv-tools_SOURCE_DIR}
+      ${spirv-tools_SOURCE_DIR}/source
```

Comment:
Hm. We already have spirv-tools_SOURCE_DIR, so instead of adding these new dirs, your #include should have the relative path from there: e.g.  "source/decoration.h" and "source/val/validation_state.h"